plugin.yaml

```diff
@@ -163,3 +203,11 @@ relationships:
 
   cloudify.aws.relationships.instance_connected_to_security_group:
     derived_from: cloudify.relationships.connected_to
+
+  cloudify.aws.relationships.instance_connected_to_load_balancer:
+    derived_from: cloudify.relationships.connected_to
+    source_interfaces:
+      cloudify.interfaces.relationship_lifecycle:
+        establish: aws.ec2.elasticloadbalancer.add_instance_to_load_balancer
+        unlink: aws.ec2.elasticloadbalancer.remove_instance_from_load_balancer
+
```

Comment:
I also want to consider the fact that sometimes we use the term "elb" sometimes "lb" and something "loadbalancer" I think we should make something consistent unless we are not talking about the same thing when we use different terms.